Index: tools/clang/tools/scan-build/ccc-analyzer
===================================================================
--- tools/clang/tools/scan-build/ccc-analyzer	(revision 155126)
+++ tools/clang/tools/scan-build/ccc-analyzer	(working copy)
@@ -434,6 +434,9 @@
 # Get the analysis options.
 my $Analyses = $ENV{'CCC_ANALYZER_ANALYSIS'};
 
+# Get the checker plugins
+my $AnalysesPlugins = $ENV{'CCC_ANALYZER_PLUGINS'};
+
 # Get the store model.
 my $StoreModel = $ENV{'CCC_ANALYZER_STORE_MODEL'};
 
@@ -649,6 +652,10 @@
       push @AnalyzeArgs, split '\s+', $Analyses;
     }
 
+    if (defined $AnalysesPlugins) {
+      push @AnalyzeArgs, split '\s+', $AnalysesPlugins;
+    }
+
     if (defined $OutputFormat) {
       push @AnalyzeArgs, "-analyzer-output=" . $OutputFormat;
       if ($OutputFormat =~ /plist/) {
